xapian-core/tests/api_backend.cc

```diff
@@ -180,6 +180,13 @@ DEFINE_TESTCASE(dbstats1, backend) {
     // 1.3.2.
     TEST_EQUAL(db.get_wdf_upper_bound(""), 0);
 
+    if (get_dbtype() == "honey") {
+	const Xapian::termcount min_unique_len = 2;
```

Comment:
How can I verify these values? I got these by setting wrong values first and then adjusting it, clearly, this is not the right away.

Reply:
That is indeed far from ideal as it wouldn't catch a bug in your implementation (though it does at least mean we have a test that fails if these values change, which is better than no test at all!)

You can find the cached database this test uses in `tests/.glass/db__etext` - and we can write a small program which just iterates all the documents in it and reports the actual highest and lowest number of unique terms, which provides a more independent confirmation:

```
~/git/xapian/xapian-core/tests$ cat uniqterms.py 
#!/usr/bin/python
from xapian import *
db = Database(".glass/db__etext")
m = 9999999
M = 0
for i in xrange(1, db.get_doccount()):
    u = db.get_unique_terms(i)
    if u > 0:
        if u < m: m = u
        if u > M: M = u
print(m, M)
~/git/xapian/xapian-core/tests$ python uniqterms.py 
(2, 272)
```

Reply:
Thanks for the script! Values match.